static/includes/dragonfly-limitations.md

```diff
@@ -1,12 +1,10 @@
-## Migration limitations at General Availability (GA)
+## Migration limitations for General Availability (GA)
```

Comment:
just wondering why we're mentioning GA in this page. Dragonfly has been GA in a while so I think we could just drop any GA reference, since it's now a product like any other.

Reply:
Maybe skip saying GA, but the limitations are valid, I suppose?

Reply:
Updated this.